eclipse/plugins/org.switchyard.tools.ui.bpmn2/META-INF/MANIFEST.MF

```diff
@@ -21,9 +21,9 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.emf.edit.ui,
  org.eclipse.emf.transaction,
  org.eclipse.bpmn2,
- org.eclipse.bpmn2.modeler.core;bundle-version="[1.1.0,1.4.0)",
- org.eclipse.bpmn2.modeler.ui;bundle-version="[1.1.0,1.4.0)",
- org.eclipse.bpmn2.modeler.runtime.jboss.jbpm;bundle-version="[1.2.3,1.4.0)",
+ org.eclipse.bpmn2.modeler.core;bundle-version="[1.1.0,1.5.0)",
```

Comment:
Yes. We hit some issues using the 1.4.0) version we had set before. Apparently "1.4.0.Final-v20170331-1748-B11" exceeds the 1.4.0 version, which meant we had to set it higher.